OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs

```diff
@@ -94,7 +99,7 @@ public static void SetTargetLines(this Actor self, List<Target> targets, Color c
 		{
 			var line = self.TraitOrDefault<DrawLineToTarget>();
 			if (line != null)
-				self.World.AddFrameEndTask(w => line.SetTargets(self, targets, color, false));
+				line.SetTargets(self, targets, color, true);
```

Comment:
I don't know :dolphin: 
I just wasn't working until I did that, to make it work I copied this from `SetTargetLine()` I think.